addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.js

```diff
@@ -26,19 +26,31 @@ const DynamicSnippetCarousel = DynamicSnippet.extend({
     //--------------------------------------------------------------------------
 
     /**
-     * Method to be overridden in child components in order to prepare QWeb
-     * options
-     * @private
+     * @override
+     */
+    _getQWebRenderOptions: function () {
+        return Object.assign(
+            this._super.apply(this, arguments),
+            {
+                interval: parseInt(this.$target[0].dataset.carouselInterval),
+            },
+        );
+    },
+    /**
+     * @deprecated
+     * @todo remove me in master, this was wrongly named and was supposed to
+     * override _getQWebRenderOptions. This is kept for potential custo in
+     * stable, although note that without hacks, calling this._super here just
+     * crashes.
```

Comment:
So this is kept in case of customisation but it's never called and calling it would break? Why keep it then? It seems that even if it were overridden it wouldn't work, and if it's incorrectly overridden since it's never called nothing will happen even if it's removed?

Reply:
Yes, @rdeodoo told me to remove it but I finally kept it :shrug: Don't like to make exception to the stable policy rule in general, even if very unlikely to break anything or even very unlikely some code relied on it. As the comment says, "calling it without hacks" would break... but you can make it work. Or even simply have some code checking the method exists. As here it was only about to keep a method, there is 0 advantage in removing it here if removed in master. So 0.000000001% risk of breaking something for a 0% gain, I preferred a 0% risk.